### PR TITLE
Authentication updates

### PR DESCRIPTION
**Pull request by dannoakes**
Authentication updates aligned with Azure AD federated identity provider guidance.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

**Review by JohanFreelancer9 — APPROVED** [review]

File: microsoft-365/managed-desktop/get-ready/prerequisites.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ Area | Prerequisite details
...
...user accounts, you must configure one of these in Azure AD Connect:<br>- Password hash synchronization<br>- Pass-through authentication<br>- An external identity provider (including Windows Server ADFS and non-Microsoft IDPs) configured to meet Azure AD integration requirements - please refer to these [guidelines](https://www.microsoft.com/en-us/download/details.aspx?id=56843). <br><br>When setting authentication options with Azure AD Connect, password writeback is also recommended. For more information, see [Password writeback](https://docs.microsoft.com/azure/active-directory/authentication/howto-sspr-writeback). <br><br>If an external identity provider is implemented, you must validate the solution:<br>- Meets Azure AD integration requirements<br>- Supports Azure AD Conditional Access, this is to enable MMD Device Compliance Policy to be configured<br>- Enables device enrollment and use of Microsoft 365 services or features required as part of Microsoft Managed Desktop <br><br>For more information on authentication options with Azure AD, see [Azure AD Connect user sign-in options](https://docs.microsoft.com/azure/active-directory/connect/active-directory-aadconnect-user-signin).
```
Comment (JohanFreelancer9) [inline microsoft-365/managed-desktop/get-ready/prerequisites.md]:
```suggestion
Authentication |	If Azure AD is not the source of primary authentication for user accounts, you must configure one of these in Azure AD Connect:<br>- Password hash synchronization<br>- Pass-through authentication<br>- An external identity provider (including Windows Server ADFS and non-Microsoft IDPs) configured to meet Azure AD integration requirements - please refer to these [guidelines](https://www.microsoft.com/download/details.aspx?id=56843). <br><br>When setting authentication options with Azure AD Connect, password writeback is also recommended. For more information, see [Password writeback](https://docs.microsoft.com/azure/active-directory/authentication/howto-sspr-writeback). <br><br>If an external identity provider is implemented, you must validate the solution:<br>- Meets Azure AD integration requirements<br>- Supports Azure AD Conditional Access, this is to enable MMD Device Compliance Policy to be configured<br>- Enables device enrollment and use of Microsoft 365 services or features required as part of Microsoft Managed Desktop <br><br>For more information on authentication options with Azure AD, see [Azure AD Connect user sign-in options](https://docs.microsoft.com/azure/active-directory/connect/active-directory-aadconnect-user-signin).
```

Comment (JohanFreelancer9) [reply microsoft-365/managed-desktop/get-ready/prerequisites.md]: Please remove the locale (en-us) from the guidelines link.

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - copy is fine but please remove the locale from the link as indicated.

**Pull request merged**